app/actions/configurations.py

```diff
@@ -1,29 +1,44 @@
 import pydantic
-from .core import PullActionConfiguration, AuthActionConfiguration
+from .core import PullActionConfiguration, AuthActionConfiguration, ExecutableActionMixin
+from .gfwclient import IntegratedAlertsConfidenceEnum, NasaViirsFireAlertConfidenceEnum
 
-
-class AuthenticateConfig(AuthActionConfiguration):
+class AuthenticateConfig(AuthActionConfiguration, ExecutableActionMixin):
     email: str
-    password: str
-
+    password: pydantic.SecretStr = pydantic.Field(..., format="password", 
+                                                  title="Password",
+                                                  description="Password for the Global Forest Watch account.")
```

Comment:
This `pydantic.SecretStr` is great. We should use it in other integrations, and maybe in any examples we add to our docs, to encourage devs to use it.